build/Packaging/AzureSiteExtension/Content/install.ps1

```diff
@@ -311,7 +327,8 @@ try
 
 	$packageNames = @($nugetPackageForFrameworkApp, $nugetPackageForCoreApp)
 	$stagingFolders = @("NewRelicPackage", "NewRelicCorePackage")
-	$newRelicInstallPaths = @("$env:WEBROOT_PATH\newrelic", "$env:WEBROOT_PATH\newrelic_core")
+	$newRelicInstallPaths = @("$env:HOME\NewRelicAgent\newrelic", "$env:HOME\NewRelicAgent\newrelic_core")
```

Comment:
@vuqtran88 as noted in my other comment, the storage root is now `%HOME%` instead of `%WEBROOT_PATH%`